src/main/java/seedu/address/logic/parser/ListModulesCommandParser.java

```diff
@@ -22,6 +22,10 @@ public ListModulesCommand parse(String userInput) throws ParseException {
         }
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_MODULE_CODE);
         String modulePrefix = argMultimap.getValue(PREFIX_MODULE_CODE).get();
+        if (modulePrefix.trim().isEmpty()) {
+            throw new ParseException(
+                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, ListModulesCommand.MESSAGE_USAGE));
+        }
```

Comment:
should this be in `ParserUtil.java`? where a `parseModulePrefix` returns the trimmed string? but it seems to work even if i have whitespace ard the moduleCode though so it is still functional

Reply:
Nice catch!